section2/reflection.md

```diff
@@ -1,15 +1,17 @@
 ## Section 2 Reflection
 
-1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
-
-2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
-
-3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
-
-4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
-
-5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
-
-6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
-
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+1. **What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?**
+  1. In the article, I liked how the Zone of Proximal Development (ZPD) was brought up. Being new to coding, I can get overwhelmed sometimes; I feel like I won't ever learn everything I need to know to be a successful developer. However, it is true that we won't be asked to do something completely unknown all at once, but rather, we will slowly challenge ourselves in ways that encourage consistent, steady progress. This is crucial to remember to avoid becoming discouraged.
```

Comment:
Yes, just venturing into one little unknown at a time. Progress is all that matters. No one ever knows it all!